src/Controls/src/Core/PlatformConfiguration/WindowsSpecific/Page.cs

```diff
@@ -49,30 +49,49 @@ public static ToolbarPlacement GetToolbarPlacement(this IPlatformElementConfigur
 
 		#region ToolbarDynamicOverflowEnabled
 
-		/// <summary>Bindable property for attached property <c>ToolbarDynamicOverflowEnabled</c>.</summary>
+		/// <summary>
+		/// Indicates whether toolbar items automatically move to the overflow menu when space is limited.
+		/// </summary>
 		public static readonly BindableProperty ToolbarDynamicOverflowEnabledProperty =
 			BindableProperty.CreateAttached("ToolbarDynamicOverflowEnabled", typeof(bool),
 				typeof(FormsElement), true);
 
-		/// <include file="../../../../docs/Microsoft.Maui.Controls.PlatformConfiguration.WindowsSpecific/Page.xml" path="//Member[@MemberName='GetToolbarDynamicOverflowEnabled'][1]/Docs/*" />
+		/// <summary>
+		/// Gets a value that indicates whether toolbar items automatically move to the overflow menu when space is limited.
+		/// </summary>
+		/// <param name="element">A page, the VisualElement that occupies the entire screen.</param>
+		/// <returns>True if toolbar items automatically move to the overflow menu when space is limited; otherwise, false.</returns>
```

Comment:
You can reference `true` and `false` using `<see langword="true"/>`. This provides better highlighting and display